data/faq/application_template.md

```diff
@@ -0,0 +1,122 @@
+coala 2017 Application Template
+===============================
+
+```
+Student Info
+------------
+
+- Name:
+- GitHub username:
+- Alternative-/Nickname:
+- Email:
+- Which country will you reside in during the summer?
```

Comment:
this was originally written for gsoc (summer for summer of code) but I guess it would be a nice idea to use the same template for all projects